packages/jest-runtime/src/RuntimeModuleNotFoundError.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as path from 'path';
+import {ModuleNotFoundError} from 'jest-resolve';
+import {Config} from '@jest/types';
+import slash = require('slash');
+
+export default class RuntimeModuleNotFoundError extends ModuleNotFoundError {
+  public hint?: string;
+  public requireStack?: Array<Config.Path>;
+  public stackTraceBuilt?: boolean;
+
+  public static buildMessage(
```

Comment:
should we just add this to `ModuleNotFoundError` directly instead of extending it?